Wikipedia/Code/YearInReviewCoordinator.swift

```diff
@@ -557,8 +597,8 @@ final class YearInReviewCoordinator: NSObject, Coordinator {
             thirdSlide = saveCountSlide
         }
 
-        if let editCountSlide = personalizedSlides.editCount {
-            fourthSlide = editCountSlide
+        if let viewCountSlide = personalizedSlides.viewCount {
+            fourthSlide = viewCountSlide
```

Comment:
One last thing! We should keep the editCount slide as the fourth slide, and insert the edits viewed count slide as a fifth slide. It might help to reference the [Figma](https://www.figma.com/design/JRTyGA3rZ0qlQ9eqh0FMhE/iOS---Wikipedia-YiR?node-id=2131-2181&node-type=canvas&t=u9Rh2aPv8fkxySZC-0) for an overview.